windows_containers/index.adoc

```diff
@@ -6,6 +6,11 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+
+Documentation for {productwinc} will be available for {product-title} {product-version} in the near future.
```

Comment:
I think we are supposed to avoid concrete forward-looking statements IIUC. If you have approval for this language from other sources, please disregard my comment :)
```suggestion
Documentation for {productwinc} is planned to be available for {product-title} {product-version} in the near future.
```